src/helpers/tokens/generate.token.js

```diff
@@ -6,7 +6,7 @@ const { Token } = models;
 const createToken = (user, secret) => {
   try {
     const token = jwt.sign({ user }, secret, {
-      expiresIn: '24h'
+      expiresIn: '2465456635435463h'
```

Comment:
all these hours for what??? @nkpremices

Reply:
I've forgotten to undo it, brother. It was for creating a token that will last for to many hours. Now I forgot to come back. 
Let me change it